inference-engine/ie_bridges/python/sample/hello_classification/hello_classification.py

```diff
@@ -16,13 +16,17 @@ def parse_args() -> argparse.Namespace:
     parser = argparse.ArgumentParser(add_help=False)
     args = parser.add_argument_group('Options')
     args.add_argument('-h', '--help', action='help', help='Show this help message and exit.')
-    args.add_argument('-m', '--model', required=True, type=str,
-                      help='Required. Path to an .xml or .onnx file with a trained model.')
+    args.add_argument('-m', '--model', required=True, type=str, help='Required. Path to an .xml or .onnx file with a trained model.')
     args.add_argument('-i', '--input', required=True, type=str, help='Required. Path to an image file.')
-    args.add_argument('-d', '--device', default='CPU', type=str,
-                      help='Optional. Specify the target device to infer on; CPU, GPU, MYRIAD, HDDL or HETERO: '
-                      'is acceptable. The sample will look for a suitable plugin for device specified. '
-                      'Default value is CPU.')
+    args.add_argument(
+        '-d',
```

Comment:
IMHO it's not user friendly way of defining argument

Reply:
I prefer a new style and it can be fixed by auto-formatter.

Reply:
@IRDonch which style do you use in OMZ ?

Reply:
The original one (which I'm pretty sure was inherited from IE samples, so it's not very surprising).

Reply:
I think we should choose a new version because we can do such changes automatically via formatter. Could you approve?

Reply:
@generalova-kate new style is not consistent with other arguments which fit into one line. Can you please check if there are other options available in auto-formatter?

Reply:
@SDxKeeper black tries to set up in one line (size 160 like for C++ code), if it can't, it set each row on a new line.

Reply:
@SDxKeeper @IRDonch I added ignoring these changes for formatter, please review one more time.